### ISSUE TITLE
PowerShell 7.2.2.1 Dependency Error Libssl

### ISSUE THREAD

**Issue by kevinevans1** [PowerShell/PowerShell#17125]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://github.com/PowerShell/PowerShell/releases)
- [X] Search the existing issues.
- [X] Refer to the [FAQ](https://github.com/PowerShell/PowerShell/blob/master/docs/FAQ.md).
- [X] Refer to [Differences between Windows PowerShell 5.1 and PowerShell](https://docs.microsoft.com/powershell/scripting/whats-new/differences-from-windows-powershell).

### Steps to reproduce

Download Latest Version Of Powershell 7
https://github.com/PowerShell/PowerShell/releases/download/v7.2.2/powershell_7.2.2-1.deb_amd64.deb

# Install the downloaded package
sudo dpkg -i powershell_7.2.2-1.deb_amd64.deb

# Resolve missing dependencies and finish the install (if necessary)
sudo apt-get install -f

### Expected behavior

```console
Powershell 7 installs on Unbuntu
```

### Actual behavior

```console
Installation Errors out due to dependency issue
```

### Error details

```console
kevin   Downloads    sudo dpkg -i powershell_7.2.2-1.deb_amd64.deb9:27
Selecting previously unselected package powershell.
(Reading database ... 294129 files and directories currently installed.)
Preparing to unpack powershell_7.2.2-1.deb_amd64.deb ...
Unpacking powershell (7.2.2-1.deb) ...
dpkg: dependency problems prevent configuration of powershell:
 powershell depends on libssl1.1 | libssl1.0.2 | libssl1.0.0; however:
  Package libssl1.1 is not installed.
  Package libssl1.0.2 is not installed.
  Package libssl1.0.0 is not installed.

dpkg: error processing package powershell (--install):
 dependency problems - leaving unconfigured
Processing triggers for man-db (2.10.2-1) ...
Errors were encountered while processing:
 powershell
```

### Environment data

```powershell
n/a
```

### Visuals

![image](https://user-images.githubusercontent.com/68919619/162413783-8803dd27-437b-4ad5-a30f-47b6eb02745b.png)

**Comment by kevinevans1** [comment]
###Adding Ubuntu Version

Distributor ID:	Ubuntu
Description:	Ubuntu Jammy Jellyfish (development branch)
Release:	22.04
Codename:	jammy

**Comment by jborean93** [comment]
I don’t know much about dpkg but doesn’t this just mean you are trying to install the dep but not all the dependencies have been met? Have you installed OpenSSL with one of the versions required?

**Comment by ctophs** [comment]
Yes, jammy went openssl 3 so the old deps of libssl 1.1.1 are not met anymore. Please provide packages in the 22.04 repository built against current jammy which is expected to be released end of April 2022

**Comment by ctophs** [comment]
22.04 LTS is tomorrow :-)

**Comment by kevinevans1** [comment]
> 22.04 LTS is ~tomorrow~ today :-)

Sweet!

**Comment by daxian-dbw** [comment]
@TravisEz13 This is on Ubuntu 22.04. Assign to you for comments.

**Comment by SteveL-MSFT** [comment]
This depends on dotnet resolving libssl3 support first https://github.com/dotnet/runtime/issues/67986

**Comment by tristanbarcelon** [comment]
@SteveL-MSFT , .Net Core sdk 6.0.202 added support for libssl3 so I was able to install it in ubuntu 22.04 without any issues. Are there plans to bump Powershell LTS release to reference this sdk version and change the debian package definition so it supports libssl3 in addition to libssl1.1?

**Comment by microhobby** [comment]
Installing the https://packages.ubuntu.com/impish/amd64/libssl1.1/download manually could be a workaround for now (at least the installation was possible)

**Comment by TravisEz13** [comment]
The SDK was bumped to 6.0.202 in 7.2.3.  We are hoping to fix the package in the next release.  https://github.com/PowerShell/PowerShell/commit/00bab545dc9c1eb4f885b0eea5b0f1647bab16b0

**Comment by TravisEz13** [comment]
This should be fixed for 7.2 now.  The next preview should have the fix as well.

**Comment by kasini3000** [comment]
```
curl -L https://gitee.com/chuanjiao10/kasini3000_agent_linux/raw/master/ubuntu2204_install_powershel.bash | /bin/bash
```

work,tested.  note: this open root password logon,
